meetings/tool/src/main/resources/validations_eu.properties

```diff
@@ -0,0 +1,3 @@
+empty_field=This field is required to be filled out
```

Comment:
@pablosanroman would you mind contributing a translation? Thanks!

Reply:
Hi @mpellicer 

empty_field= Eremu hau nahitaez bete behar da
empty_checkbox= Egiaztapen-lauki honek markatuta egon behar du
maxlenght_field= Balioak gainditzen du baimendutako gehieneko tamaina